Automattic/jetpack
# Boost: Fix UI issues

Fix various UI bugs discovered during 1.9 beta test 1.

## Proposed changes:

* Add missing capitalization for speed score label;
* Fix tooltips positioning issues on getting started page;
* Fix concatenated exclude lists showing description when nothing was excluded.

### Other information:


## Jetpack product discussion



pedxs5-eK-p2#comment-107

## Does this pull request change what data or activity we track or use?



no

## Testing instructions:






* Install beta 1.9;
* Check to see if the issues listed in pedxs5-eK-p2#comment-107 are present.
